src/test/java/com/powsybl/network/conversion/server/NetworkConversionTest.java

```diff
@@ -127,10 +137,21 @@ class NetworkConversionTest {
     private S3Client s3Client;
 
     @BeforeEach
-    void setup() {
+    void setup() throws IOException {
+        fileSystem = Jimfs.newFileSystem(provideJimfsUnixConfigurationWithPosixFileAttributes());
```

Comment:
```suggestion
        fileSystem = Jimfs.newFileSystem(Configuration.unix());
```
That's what we use in powsybl-core and powsybl-case-server